lib/rules/declaration-block-no-redundant-longhand-properties/__tests__/index.mjs

```diff
@@ -301,6 +301,54 @@ testRule({
 			description: 'explicit border-inline test',
 			message: messages.expected('border-inline'),
 		},
+		{
+			code: 'div { overflow-y: visible; overflow-x: hidden; }',
+			fixed: 'div { overflow: hidden visible; }',
+			description: 'explicit overflow test',
+			message: messages.expected('overflow'),
+		},
+		{
+			code: 'div { overscroll-behavior-y: contain; overscroll-behavior-x: auto; }',
+			fixed: 'div { overscroll-behavior: auto contain; }',
+			description: 'explicit overscroll-behavior test',
+			message: messages.expected('overscroll-behavior'),
+		},
+		{
+			code: 'div { column-gap: 20px; row-gap: 10px; }',
+			fixed: 'div { gap: 10px 20px; }',
+			description: 'explicit gap test',
+			message: messages.expected('gap'),
+		},
+		{
+			code: 'div { justify-content: center; align-content: end; }',
+			fixed: 'div { place-content: end center; }',
+			description: 'explicit place-content test',
+			message: messages.expected('place-content'),
+		},
+		{
+			code: 'div { justify-items: center; align-items: end; }',
+			fixed: 'div { place-items: end center; }',
+			description: 'explicit place-items test',
+			message: messages.expected('place-items'),
+		},
+		{
+			code: 'div { justify-self: center; align-self: end; }',
+			fixed: 'div { place-self: end center; }',
+			description: 'explicit place-self test',
+			message: messages.expected('place-self'),
+		},
+		{
+			code: 'a { scroll-margin-left: 10px; scroll-margin-right: 10px; scroll-margin-top: 20px; scroll-margin-bottom: 30px; }',
```

Comment:
minor nit

always use different values
i.e. we couldn't tell if `right` and `left` were badly positioned

Reply:
Thanks! I also adjusted the original `margin` test that I got this line from.